From 0036cca83011affbe91ec5434ced5041be3e5d4a Mon Sep 17 00:00:00 2001
From: Pawel Osciak <posciak@chromium.org>
Date: Mon, 11 Jul 2016 18:25:17 +0900
Subject: [PATCH] CHROMIUM: v4l: Add VP9 low-level decoder API controls.

Signed-off-by: Pawel Osciak <posciak@chromium.org>

[fbuergisser@chromium.org: original patch modified as h264 headers were moved
 to other header files which cause a failure when applying the original patch.
 Keeping part of the original patch for context.]

[acourbot@chromium.org: moved controls to a "legacy" VP9 files as we
will merge the upstream ones.]

[wenst@chromium.org: dropped all the legacy controls, only keeping the
pixel format macro. This should later be merged directly into a VP9 uAPI
backport patch.]
---
 include/uapi/linux/videodev2.h              |   2 +
 2 files changed, 164 insertions(+)
 create mode 100644 include/uapi/linux/media/vp9-ctrls-legacy.h

diff --git a/include/uapi/linux/videodev2.h b/include/uapi/linux/videodev2.h
index 41e0808..2d8f794 100644
--- a/include/uapi/linux/videodev2.h
+++ b/include/uapi/linux/videodev2.h
@@ -639,6 +639,7 @@ struct v4l2_pix_format {
 #define V4L2_PIX_FMT_VP8      v4l2_fourcc('V', 'P', '8', '0') /* VP8 */
 #define V4L2_PIX_FMT_VP8_FRAME v4l2_fourcc('V', 'P', '8', 'F') /* VP8 parsed frames */
 #define V4L2_PIX_FMT_VP9      v4l2_fourcc('V', 'P', '9', '0') /* VP9 */
+#define V4L2_PIX_FMT_VP9_FRAME v4l2_fourcc('V', 'P', '9', 'F') /* VP9 parsed frames */
 #define V4L2_PIX_FMT_HEVC     v4l2_fourcc('H', 'E', 'V', 'C') /* HEVC aka H.265 */
 #define V4L2_PIX_FMT_H264_SLICE v4l2_fourcc('S', '2', '6', '4') /* H264 parsed slices */
 
-- 
2.26.2

